.storybook/preview.js

```diff
@@ -0,0 +1,24 @@
+import * as NextImage from 'next/image'
+import { WalletProvider } from '../components/WalletProvider'
+import { theme } from '../theme'
+
+// de-optimize Image component
+const OriginalNextImage = NextImage.default
+Object.defineProperty(NextImage, 'default', {
+  configurable: true,
+  value: (props) => <OriginalNextImage {...props} unoptimized />,
+})
+
+export const parameters = {
+  chakra: {
+    theme,
+  },
+}
+
+export const decorators = [
```

Comment:
[`decorators`](https://storybook.js.org/docs/react/writing-stories/decorators) are essentially wrappers around our stories. `decorators` defined in this file are global, meaning every story will be wrapped by them when rendered.